Explorer/Assets/Scripts/ECS/Unity/Transforms/Components/TransformComponent.cs

```diff
@@ -63,6 +64,18 @@ public void SetTransform(Vector3 localPosition, Quaternion localRotation, Vector
             Cached.WorldRotation = Transform.rotation;
         }
 
+        public void SetTransform(Transform transform)
+        {
+            SetTransform(transform.localPosition, transform.localRotation, transform.localScale);
+        }
```

Comment:
can I ask you of small optimization? Can you please change setting position and rotation inside `SetTransform `method to             `Transform.SetLocalPositionAndRotation(localPos, localRot);`

Reply:
- I think Explorer/Assets/DCL/SDKComponents/Billboard makes a sense
- Good suggestion
- Like what for example? Do you mean put ':' into it?

Reply:
> can I ask you of small optimization? Can you please change setting position and rotation inside `SetTransform ` method to unity built-in `Transform.SetLocalPositionAndRotation(localPos, localRot);`

Nice mention, I added it

Reply:
thanks for applying changes. Regarding the title,  ':' is enought, but it could also specify that feature is related to the SDK/Scene (similar to the folder - **Feat: billboard sdk component support**). But it is super cosmetical, so you can keep it as it is 🙂